.storybook/preview.js

```diff
@@ -12,21 +12,92 @@ SyntaxHighlighter.registerLanguage('scss', scss);
 
 export const parameters = {
   options: {
-    storySort: {
-      order: [
-        'Welcome',
-        'Getting Started',
-        ['Development Guide', ['Installation', ['*']]],
-        'Releases',
-        ['v0.5.x', 'v0.4.x', 'v0.3.x', 'v0.2.x', 'v0.1.x', 'v0.0.x'],
-        'Atoms',
-        'Molecules',
-        'Organisms',
-        'Templates',
-        'Utilities',
-        ['Directives', ['Docs', '*']],
-      ]
-    },
+    storySort: (story1, story2) => {
+      /* 
+        Sort function to sort stories based on a config file.
+        {storiesOrder} is the order in which we want stories to appear.
+        Any stories that aren't explicitly listed will appear at the end of list in alphabetical order.
+        Modifiers: 
+          {'*'} - every nested element
+          {'rest-reverse'} - rest of not listed elements appear in reverse alphabetical order
+      */
+      const storiesOrder = {
+        // Keys must be in camelCase
+        welcome: {},
+        team: {},
+        gettingStarted: {
+          developmentGuide: {
+            installation: {},
+          },
+        },
+        releases: {
+          '*': {
+            'rest-reverse': {
+              'rest-reverse': {
+                changeLog: {}
+              },
+            }
+          },
+        },
+        atoms: {
+          '*': {
+            common: {}
+          },
+        },
+        molecules: {
+          '*': {
+            common: {}
+          },
+        },
+        organisms: {
+          '*': {
+            common: {}
+          },
+        },
+        templates: {
+          '*': {
+            common: {}
+          },
+        },
+        utilities: {
+          directives: {
+            docs: {}
+          },
+        },
+        contributingGuide: {}
+      };
+      const hasKey = (obj, key) => Object.hasOwn(obj, key);
+      const compareAlphabetical = (a, b) => a.localeCompare(b, { numeric: true });
```

Comment:
maybe `a` and `b` it's not the best name :D maybe `previous` and `next` or something like that